ansible/pbTestScripts/vagrantPlaybookCheck.sh

```diff
@@ -371,26 +371,33 @@ startVMPlaybookWin()
 
 destroyVM()
 {
-	local OS=$1
-	echo "Destroying the $OS Machine"
-	echo === `date +%T`: showing global status before pruning:
-	vagrant global-status
-	free
-	echo === showing global status while pruning:
-	vagrant global-status --prune
-	echo === Determining VM to destroy:
-	VM_TO_DESTROY=`vagrant global-status --prune | grep $OS | awk "/${gitFork}-${gitBranch}/ { print \\$1 }"`
-	if [ ! -z "$VM_TO_DESTROY" ]; then
-	  echo === Destroying VM with id $VM_TO_DESTROY
-	  vagrant destroy -f $VM_TO_DESTROY
+	if [[ "$retainVM" == false ]]; then
+		echo "Destroying the VM"
```

Comment:
Do we really need this echo statement given there's another more descriptive one two lines down?

Reply:
I added this as a debug line, forgot to remove it :L